staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -214,9 +214,9 @@ func Record(req *http.Request, requestInfo *request.RequestInfo, component, cont
 	}
 	scope := CleanScope(requestInfo)
 	if requestInfo.IsResourceRequest {
-		MonitorRequest(req, strings.ToUpper(requestInfo.Verb), requestInfo.APIGroup, requestInfo.APIVersion, requestInfo.Resource, requestInfo.Subresource, scope, component, contentType, code, responseSizeInBytes, elapsed)
+		MonitorRequest(req, strings.ToUpper(req.Method), requestInfo.APIGroup, requestInfo.APIVersion, requestInfo.Resource, requestInfo.Subresource, scope, component, contentType, code, responseSizeInBytes, elapsed)
```

Comment:
does this mean we cannot see the difference between the logical verbs LIST and GET anymore when filtering by GVK ?

Reply:
Could we technically do the opposite: use the logical RequestInfo verb instead?

Reply:
> does this mean we cannot see the difference between the logical verbs LIST and GET anymore when filtering by GVK ?

good catch. Fortunately we have enough information to fix it (scope) so we can actually do the correct thing. Will fix.

> Could we technically do the opposite: use the logical RequestInfo verb instead?

Technically we can. But that would labels for majority of metrics, so it would be quite risky in my opinion.

Reply:
BTW - it seems that LIST isn't really handled here too:
https://github.com/kubernetes/kubernetes/blob/d6035f3e0d79dd05628ef42231beae97806a06ad/staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go#L162